mongo/mgo_job_store.go

```diff
@@ -0,0 +1,198 @@
+package mongo
```

Comment:
yes I agree - I had named it with the mgo prefix to make things clearer to me while the old job_store class, which used a Map rather than Mongo, was still there (but I've deleted that now). - prefix now removed